### PR TITLE
Add Qubic Project plus qubic draft one docs

### PR DESCRIPTION
**Pull request by NelsonPython**

### CODE REVIEW

File: qubic-project/0.1/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+Qubic is a protocol that specifies IOTA's solution for oracle machines, smart contracts, and outsourced computations.  As of February 2019, development is underway.  Please see the development [roadmap](https://qubic.iota.org/roadmap).
+
+## Components
+
+- [Qubic software](0.1\qubic\README.md)
+
+- [**Q**ubic **C**omputation **M**odel (QCM)](https://github.com/iotaledger/qcm)
+
+- **I**ota **C**ontrol agen**T** (ICT) - light-weight node, called a “Q-Node”
```
Comment (mikrohash) [inline qubic-project/0.1/README.md]: Q-node refers to any node providing qubic computations as a service. This is a higher layer that could run on top of IRI/cIRI or Ict. IOTA nodes such as IRI and Ict define the IOTA protocol. Qubic is built on top of / utilizes the IOTA protocol.

**Comment by l3wi** [comment]
@mikrohash Could you please have another look over to make sure this solves your comments?

File: qubic-project/0.1/README.md (hunk 1 above)
Comment (NelsonPython) [reply qubic-project/0.1/README.md]: This pull request is a placeholder for docs that are being written based on design decisions by the Omega team as work is underway.  I believe the Omega team leadership should approve it before it is published.

**Comment by jakubcech** [comment]
Will this be linked to only? Right now it doesn't have projects.md, which means it won't be in the floating  landing page menu, burger menu, nor the footer. The issue with adding it there now is that we don't have anymore room at the floating landing page menu (someone took the last spot with App blueprints 😭 ). So we'll need to rethink the highest level structure a bit before merging if we want to it be navigate-able to.

**Comment by NelsonPython** [comment]
This information is outdated

**Pull request closed without merging**